<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtendedNumericInput</class>
 <widget class="QFrame" name="ExtendedNumericInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="inputTypeButton">
     <property name="text">
      <string/>
     </property>
     <property name="arrowType">
      <enum>Qt::DownArrow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="single">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QSpinBox" name="spinBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="range">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>from</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="minSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>to</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="maxSpinBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="biomimal">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Number of tries:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="numSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Probability:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="probSpinBox"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
